### PR TITLE
[release-v1.9] Point to Eventing midstream

### PR DESCRIPTION
**Pull request by pierDipi**
This is to pick up the autoscaler/scheduler changes

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -164,3 +164,5 @@ require (
 )
 
 replace github.com/cloudevents/sdk-go/sql/v2 => github.com/cloudevents/sdk-go/sql/v2 v2.0.0-20220930150014-52b12276cc4a
+
+replace knative.dev/eventing => github.com/openshift-knative/eventing v0.99.1-0.20230831170232-cf89b7eadfce
```
Comment (matzew) [inline go.mod:168]: check

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Comment by matzew** [comment]
@pierDipi  what about porting to other branches?

**Comment by pierDipi** [comment]
I'm working on that

**Pull request merged**